database/usersDataManager.ts

```diff
@@ -0,0 +1,71 @@
+import { DB_CLIENT } from "./dbConfig";
+
+const USERS_TABLE: string = "boba-buddies-users";
```

Comment:
im keeping track of the slack users in the boba-buddies channel for the pr i will make on generating pairs to account for people who join halfway thru the semester